Makefile

```diff
@@ -0,0 +1,29 @@
+VERSION ?= v0.8.3
```

Comment:
move to tests/

Reply:
I think it's OK to require the `VERSION` variable when running download command

no need for a default value here - it will be passed down from CI